### PR TITLE
Fix broken change content version

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/widget/versionChannel.coffee [hunk 1]
```diff
@@ -8,13 +8,17 @@ widgetChannel.bind 'ready', (view) ->
...
         collection.set response
         collectionName = collection.get('collection_name')
+        versionSelectView = new VersionSelectView(
+          currentVersion: view.options.multiVersion
+          domContainer: view.$el.find('#version-selectbox')
+        )
         for version of collection.get(collectionName)
           versionElement = new VersionviewModel
           versionElement.set collection.get(collectionName)[version]
           new VersionView(
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/widget/versionChannel.coffee]: Move the creation of versionElement to VersionSelectView to ensure that versionSelectView.$el exists.

**Comment by itkg-ngilain** [comment]
@itkg-nanne : update done

**Comment by itkg-ngilain** [comment]
@itkg-nanne : option reduce used

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**